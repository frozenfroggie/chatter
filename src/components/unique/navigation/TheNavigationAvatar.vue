<template>
  <div class='avatarContainer'>
    <icon name="user-circle" class='avatar' scale="2.5"></icon>
    <div :class='{online: true}'></div>
  </div>
</template>

<script>
</script>

<style scoped>
.avatarContainer {
  position: relative;
  grid-area: avatar;
  justify-self: center;
  align-self: center;
  width: 40px;
  height: 40px;
  border-radius: 50%;
}
.avatar {
  width: 40px;
  height: 40px;
  object-fit: cover;
  color: white;
  opacity: .4;
}
.online {
  position: absolute;
  bottom: 0px;
  right: 0px;
  height: 6px;
  width: 6px;
  border: 3px solid rgb(38, 49, 66);
  border-radius: 50%;
  background-color: rgb(66, 183, 42);
}
</style>
