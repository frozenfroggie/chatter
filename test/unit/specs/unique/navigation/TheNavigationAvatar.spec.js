import { shallowMount, mount } from '@vue/test-utils'
import TheNavigationAvatar from '@/components/unique/navigation/TheNavigationAvatar'

describe('TheNavigationAvatar', () => {
  it('matches snapshot', () => {
    const wrapper = mount(TheNavigationAvatar, {
      stubs: ['icon']
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
