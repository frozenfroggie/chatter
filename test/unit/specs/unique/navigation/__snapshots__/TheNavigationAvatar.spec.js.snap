// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheNavigationAvatar matches snapshot 1`] = `
<div class="avatarContainer">
    <!---->
    <div class="online"></div>
</div>
`;
